x-pack/plugins/cases/public/components/custom_fields/toggle/get_eui_table_column.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import type { EuiTableComputedColumnType } from '@elastic/eui';
+import { EuiSwitch } from '@elastic/eui';
+
+import type { CaseUI } from '../../../containers/types';
+import { CustomFieldTypes } from '../../../../common/types/domain';
+import { getEmptyTagValue } from '../../empty_value';
+
+export const getEuiTableColumn = ({
+  key,
+  label,
+}: {
+  key: string;
+  label: string;
+}): EuiTableComputedColumnType<CaseUI> => ({
+  name: label,
+  render: (theCase: CaseUI) => {
```

Comment:
I think the whole render function can be simplified as:

```

const customField = theCase.customFields.find((element) => element.key === key && element.value != null); <--- we need the null check here. It can be set through the API.

const isChecked = Boolean(customField?.value)

return (
      <EuiSwitch
        data-test-subj={`toggle-custom-field-column-view-${key}`}
        label={''}
        checked={isChecked}
        onChange={() => {}}
        compressed
      />
    );
```